### DISCUSSION TITLE
TX16S MK2: Potis not longer available in global functions after upgrade to 2.10.0

### DISCUSSION THREAD

**Discussion by FearlessFlyer** [EdgeTX/edgetx#5065] [General]
Hello,
I've just upgraded to 2.10.0 and now my potis are not longer available at global functions.

On the System-Page at the Hardware-Tab, I have defined poti S1 as ''Vol" to set the volume of the speaker and poti S2 as "LCD" to set the brightness of the screen.
This worked fine with the older firmware 2.9. After upgrade to 2.10.0 (german language) the potis "Vol" and "LCD" are not longer available as switch/source at the global-functions-screen.

Edit: at the Model-Page, my potis are available at the logical-switches-screen as V1 but not at the special-functions-screen. 

What have I done wrong? Or is it a bug?
What can I do? Any ideas? Thanks in advance.

**Comment by pfeerick** [comment]
Check if they are configured in Radio Settings -> Hardware -> Pots (S1/S2 should be Pot with detent). They will never be available as the first column (trigger) as they are not switches, but should be available in the value (source) field.

**Reply by FearlessFlyer** [reply]
Thanks for your fast reply.
Yes, they are both listed correctly at the pots-page as Pot with detent ("Poti mit Raste" in german).
That's where I gave them their names "Vol" and "LCD".

Edit: the 6-Position-switch is available at the global functions. Only the two pots are missing.

BTW: the problem occures in companion as well as on the transmitter.

**Comment by keithluneau** [comment]
Try setting it like this:

Trigger - ON
Function - Backlight (or Volume)
Value - S1 (or S2)
Enable - "Switched on"

That works for me in 2.10.0

**Reply by FearlessFlyer** [reply]
This works well. Thank you very much!
